winbuild/pthread.h

```diff
@@ -80,12 +80,16 @@
 #define PTHREAD_CANCELED ((void *)(intptr_t)0xDEADBEEF)
 
 #define PTHREAD_ONCE_INIT 0
-#define PTHREAD_MUTEX_INITIALIZER {(void*)-1,-1,0,0,0,0}
+#define PTHREAD_MUTEX_CS_INITIALIZER {(void*)-1,-1,0,0,0,0}
```

Comment:
nit: probably shouldn't make this visible with the prefix `PTHREAD_`.  I'd suggest either folding it into the PTHREAD_MUTEX_INITIALIZER and not defining a separate macro, or giving it an underscore prefix eg: `_PTHREAD_MUTEX_CS_INITIALIZER`